compiler/src/iree/compiler/Utils/ElementPackingUtils.cpp

```diff
@@ -99,6 +99,11 @@ Value calculateStorageElementCountInBytes(Location loc,
     }
   }
 
+  // make sure the last dimension is byte aligned.
```

Comment:
style: proper punctuation (here and elsewhere) in comments: https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar